shared/helpers/abi.js

```diff
@@ -0,0 +1,27 @@
+const abi = require('web3-eth-abi')
+
+abiUtil = {
+
+  stringifyParams: (params) => {
+    return params.map(p => {
+      if (Array.isArray(p)) {
+        return abiUtil.stringifyParams(p)
+      }
+      else {
+        if (typeof(p) === 'boolean') {
+          return p
+        }
+        else {
+          return `${p}`
+        }
+      }
+    })
+  },
+
+  encodeFunctionCall: (funcSignature, types, params) => {
```

Comment:
We are following this convention

```suggestion
  encodeFunctionCall(signature, types, params) {
```